docs/interactions.rst

```diff
@@ -266,3 +266,23 @@ Shown below is an example of a simple command running in a cog:
             await interaction.response.send_message("Hello I am a slash command in a cog!")
 
 Context menu commands can also be made in cogs using the :meth:`nextcord.user_command` or :meth:`nextcord.message_command` decorators.
+
+
+Default Guild Ids
+-----------------
+
+If you would like for all of your application commands to have the same guild ids unless explicitly stated, then you can set the ``default_guild_ids`` keyword argument in :class:`nextcord.ext.commands.Bot` and :class:`nextcord.ext.commands.AutoShardedBot`!
```

Comment:
This works with Client too, right?

```suggestion
If you would like for all of your application commands to have the same guild ids unless explicitly stated, then you can set the ``default_guild_ids`` keyword argument in :class:`~nextcord.Client`, :class:`~nextcord.ext.commands.Bot`, or :class:`~nextcord.ext.commands.AutoShardedBot`!
```

Reply:
It does, but it looks like the rest of the examples use `ext.commands` instead of the main library.